src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -802,6 +826,7 @@ ExportQueryExecutionTrees::computeResultAsQLeverJSON(
   jsonSuffix["runtimeInformation"]["query_execution_tree"] =
       nlohmann::ordered_json(runtimeInformation);
   jsonSuffix["resultsize"] = resultSize;
+  jsonSuffix["sent"] = numBindingsYielded;
```

Comment:
Since `resultsize` can be large than the number of bindings exported, I think it's useful to have both values in the JSON, while I don't see that it does any harm.

Reply:
It does no harm, but I don't think qlever-ui will ever use the value of `sent`, when it can just use [`Array.length`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/length)